src/global_scheduler/global_scheduler.cc

```diff
@@ -339,11 +336,9 @@ void local_scheduler_table_handler(DBClientID client_id,
   /* Extract global scheduler state from the callback context. */
   GlobalSchedulerState *state = (GlobalSchedulerState *) user_context;
   ARROW_UNUSED(state);
-  char id_string[ID_STRING_SIZE];
-  LOG_DEBUG(
-      "Local scheduler heartbeat from db_client_id %s",
-      ObjectID_to_string((ObjectID) client_id, id_string, ID_STRING_SIZE));
-  ARROW_UNUSED(id_string);
+  std::string id_string = client_id.hex();
+  LOG_DEBUG("Local scheduler heartbeat from db_client_id %s",
+            id_string.c_str());
```

Comment:
What's the plan for these logging macros down the road? E.g., Arrow uses something more compatible with C++ style logging, which would be nice here so we could avoid the `.c_str` and not have to worry so much about string formatting.

Reply:
Yeah, the code is already in ray/util/logging.h and the various components should be modernized as we go along.